dipy/reconst/multi_voxel.py

```diff
@@ -61,6 +61,33 @@ def __getitem__(self, index):
         else:
             return item
 
+    def predict(self, *args, **kwargs):
+        """
+        Predict for the multi-voxel object using each single-object's
+        prediction API, with S0 provided from an array.
+        """
+        if not hasattr(self.model, 'predict'):
+            msg = "This model does not have prediction implemented yet"
+            raise NotImplementedError(msg)
```

Comment:
@Garyfallidis If a model does not have a predict method a NotImplementedError will be raised here.